pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+For a PHEP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate PyHC projects unduly. Finally, a proposed enhancement must be "pythonic" in order to be accepted by the community. (However, "pythonic" is an imprecise term; it may be defined as whatever is acceptable to the community. This logic is intentionally circular.)
+
+The editors must post oronouncements of PHEP resolution to the PyHC mailing list.
+
+Once a PHEP has been accepted, the reference implementation, if applicable, must be completed. When the reference implementation is complete and incorporated into a PyHC project's source code repository, the status will be changed to "Final". PHEPs which do not require a reference implementation move immediately to "Final" upon acceptance.
+
+If a PHEP requires additional design and interface feedback before committing to long term stability, it must remain in "Draft". Alternatively, its scope can be reduced (e.g. by deferring some features to later PHEPs).
+
+If no progress is being made on the PHEP, the editor should consult the author. The author may resume work on the PHEP, [transfer it to another author](transferring-phep-ownership), or withdraw it. If the author takes none of these actions, the editor may choose to bring the PHEP to the community for approval or rejection in its current form.
+
+If a PHEP fails to find consensus, the author may choose to modify it for reconsideration in the future. The PHEP may also be "Withdrawn" if the PHEP author themself has decided that the PHEP is actually a bad idea, or has accepted that a competing proposal is a better alternative. A PHEP can also be "Rejected" if it fails to find consensus and the author is not willing or able to modify it. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
+
+When a PHEP changes state, the PHEP text must be updated accordingly. In addition to updating the Status field, at the very least the Resolution header should be added with a direct link to the minutes of the telecons and meetings where a decision on the PHEP was made.
+
+Final PHEPs can also be replaced by a different PHEP, rendering the original obsolete.
+
+The possible paths of the status of PHEPs are as follows:
+
+![PHEP status flow diagram](phep-0001/process_flow.svg)
+
+While not shown in the diagram, "Accepted" PHEPs may technically move to "Rejected" or "Withdrawn" even after acceptance. This will only occur if the implementation process reveals fundamental flaws in the design that were not noticed prior to acceptance of the PHEP.
+
+Upon reaching "Final", "Withdrawn", or "Rejected" status, the editors update the preamble and merge the PR into `main`.
```

Comment:
We also need to assign a DOI at this point. Zenodo will assign dois to a GitHub repo when a release is made, but it refers to the _entire_ repo, and we want to refer to a particular file at a particular commit hash in the repository. Open to suggestions

Reply:
We have a (somewhat flaky) script in PlasmaPy's repo that converts PLEPs into pdfs...I think using pandoc.

Reply:
@namurphy are you suggesting we could get DOIs by submitting PDF versions of PHEPS?

Reply:
I suggest capturing mechanics of DOI handling in #24 although I will put some text here on the need for a DOI (thus not resolving this conversation yet).

Reply:
@namurphy friendly reminder to reply about these DOI PDFs; in #24 sounds like.

Reply:
Do we mint a DOI for Rejected and Withdrawn PHEPs or only Accepted? I'm leaning towards all.

Reply:
Yeah I'd say all, to maintain a complete record.

Reply:
Text regarding DOI [here](https://github.com/jtniehof/standards/blob/0cd4f81ca4dd6b498606aadb3bbe451bd0031b9b/pheps/phep-0001.md?plain=1#L145); resolve if this is captured (outside of implementation details).